[feature #115585127] Add About Us & Contact Us page

andela-joyebanji opened the pull request:
Update routes file to include routes for about us and contact us page
Create the about us and contact us page
Update navbar file to include link to about us and contact us page
Update HomeController to include methods to handle about us and contact us page request

Commit:
[feature #115585127] Add About Us & Contact Us page

Update routes file to include routes for about us and contact us page
Create the about us and contact us page
Update navbar file to include link to about us and contact us page
Update HomeController to include methods to handle about us and contact us page request

andela-anandaa commented on app/Http/Controllers/HomeController.php:
```diff
@@ -33,4 +33,14 @@ public function homepage()
         return view('welcome', compact('latestRestaurants', 'featuredRestaurants'));
     }
+
+    public function aboutus()
```
Use _camelCase_.

andela-anandaa commented on resources/views/aboutus.blade.php:
```diff
@@ -0,0 +1,28 @@
+@extends('dashboard.registration')
```
Extend the `layouts.inner` template.

Commit:
Add default value of -1 to restaurant_id column in add_restaurant_id_col_to_restaurants file
Modify BookingTest to include newly added bookings table columns
Update the url and see value to the newly created values
Comment test cases in RestaurantGallery that are not yet implemented

Commit:
Applied fixes from StyleCI

Commit:
[Chore] Change extend layout

Change the naming of about and contactus method to camelCase in HomeController
The aboutus and contactus page now extend the layouts.inner layout
Add new css file inner.css, for styling inner pages
